### DISCUSSION TITLE
Google Page Speed Insights always recommends to "Reduce initial server response time". What can I do to fix this?

### DISCUSSION THREAD

**Discussion by lumenwrites** [vercel/community#144] [Help] [answered]
Hi! I've been using Vercel for awhile now and deployed a few projects with it.

When I analyze my projects using Google PageSpeed Insights, it almost always says that I should "Reduce initial server response time", and says that it could save 1-2 seconds of loading time, which seems significant.

Most of my projects are very simple, and all they do on the backend is use prisma to fetch some data (not a lot) from an AWS RDS database, hosted in the same region (`us-east-1`, N.Virginia)  as vercel's [default region](https://vercel.com/docs/concepts/functions/regions#default-region). Some examples: [1](https://pagespeed.web.dev/report?url=https%3A%2F%2Fnexy.io%2F), [2](https://pagespeed.web.dev/report?url=https%3A%2F%2Fevertweet.app%2F), [3](https://godotassets.io/).

This happens even to very simple static websites like [this one](https://pagespeed.web.dev/report?url=https%3A%2F%2Frpgadventures.io%2F) and [this one](https://pagespeed.web.dev/report?url=https%3A%2F%2Flumenwrites.dev%2F).

Why is this happening, is there anything I can do to fix this?

**Comment by Pieparker** [comment, marked as answer]
Hey there! I can see that [a similar discussion was posted to the Next.js subreddit](https://www.reddit.com/r/nextjs/comments/rj32sz/google_page_speed_insights_always_recommends_to/). I'm glad you were able to get some advice!

**Comment by michaelmika** [comment]
I am also having troubles with vercel with server response times of more than 6 seconds on some pages. As DNS I am using Route 53 as suggested in the subreddit. Is the free tier limited? I am experiencing it mostly on the first request of the day/hour. It seems that the server needs to reactivate very slowly. However, my project has many dynamic routes that are not requested this often, so the extreme delay happens nearly on every request. Any suggestion on how to improve it?

**Reply by Pieparker** [reply]
You may want to [check this out](https://github.com/vercel/community/discussions/307#discussioncomment-2152407)!
